jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/Request.java

```diff
@@ -1546,8 +1546,8 @@ void onRequest(org.eclipse.jetty.server.Request coreRequest)
         _uri = coreRequest.getHttpURI();
 
         _pathInContext = _context.getContextHandler().isCanonicalEncodingURIs()
-            ? coreRequest.getPathInContext()
-            : URIUtil.decodePath(coreRequest.getPathInContext());
+            ? org.eclipse.jetty.server.Request.getPathInContext(coreRequest)
+            : URIUtil.decodePath(org.eclipse.jetty.server.Request.getPathInContext(coreRequest));
```

Comment:
Store `Request.getPathInContext(request)` in a local variable to avoid calling it twice.